content/lessons/chapter-4/address-1.tsx

```diff
@@ -46,30 +46,30 @@ const javascript = {
 const python = {
   program: `print("KILL")`,
   defaultFunction: {
-    name: 'find_hash',
-    args: ['nonce'],
+    name: 'hash_compressed',
+    args: ['compressed'],
   },
-  defaultCode: `from hashlib import sha256
-
-# Create a program that finds a sha256 hash starting with 5 zeroes.
-# To submit your answer, print it to the terminal using print().
-
+  defaultCode: `import hashlib
+# Get the sha256 digest of the compressed public key.
+# Then get the ripemd160 digest of that sha256 hash
+# Return 20-byte array
 # Type your code here
 `,
   validate: async (answer) => {
-    if (!answer.startsWith('00000')) {
-      return [false, 'Hash must start with 5 zeroes.']
+   
+    if (len(answer)!==20) {
```

Comment:
this should be `if (answer.length !== 20) {` remember even though this is checking the result of python code we are still using JS! Also if you are failing to compile remember that you can always run `yarn build` or `yarn lint` to check code quality

Reply:
Thanks @benalleng it worked!!